bin/tests/http.rs

```diff
@@ -0,0 +1,64 @@
+use common::AgentSettings;
+pub use common::*;
+use std::fs::File;
+use std::io::Write;
+use std::sync::{Arc, Mutex};
+use tempfile::tempdir;
+
+mod common;
+
+#[tokio::test]
+#[cfg_attr(not(feature = "integration_tests"), ignore)]
+async fn test_http_buffer_size() {
+    let dir = tempdir().unwrap().into_path();
+    let file_path = dir.join("test.log");
+    let mut file = File::create(&file_path).expect("Couldn't create temp log file...");
+
+    let call_counter = Arc::new(Mutex::new(0));
+    let counter = call_counter.clone();
+    let (mock_server, _, shutdown_handle, address) =
+        common::start_ingester(Box::new(move |body| {
+            // The agent picks up some of its own log messages and sends them to the mock ingest service
```

Comment:
Hmm, it shouldn't be picking up it's own lines. We work around this in other tests by setting the exclusion/inclusion rules to be very specific using the AgentSetting struct at https://github.com/logdna/logdna-agent-v2/blob/fa5726f4d7acc1f8b19044158459c2daf0692a60/bin/tests/common.rs#L132.

I wonder why the same isn't working here...